cmd/lakectl/cmd/annotate.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+const annotateTemplate = `{{  $val := .Commit}}
+{{ $.Object|ljust 15}} {{ $val.Committer|ljust 20 }} {{ $val.Id | printf "%.16s"|ljust 20 }} {{ $val.CreationDate|date }}  {{ $.CommitMessage |ljust 30 }}
+`
+
+var annotateCmd = &cobra.Command{
+	Use:   "annotate <path uri>",
+	Short: "List entries under a given path, annotating each with the latest modifying commit ",
+	// Long:    "Show who created the latest commit to an object or objects in a give path",
+	Aliases: []string{"blame"},
+	Args:    cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		pathURI := MustParsePathURI("path", args[0])
+		client := getClient()
+		amount := 1
+		pfx := api.PaginationPrefix(*pathURI.Path)
+		res, err := client.ListObjectsWithResponse(cmd.Context(), pathURI.Repository, pathURI.Ref, &api.ListObjectsParams{Prefix: &pfx})
```

Comment:
suggestion: should assign context to a variable - because we use it later

Reply:
Done